app/classifier/tasks/drawing-task-details-editor.cjsx

```diff
@@ -4,6 +4,10 @@ AutoSave = require '../../components/auto-save'
 module.exports = React.createClass
   displayName: 'DrawingTaskDetailsEditor'
 
+  propTypes:
```

Comment:
This is generally unnecessary in a component with such a specific instance (very handy in more reusable ones, though). If you do define `propTypes`, I'd define all of them (this is missing `toolIndex`).